sample_code/ep025/rev03/flake8_named_arguments.py

```diff
@@ -20,17 +20,17 @@ def __init__(self) -> None:
 
     def visit_Call(self, node: ast.Call) -> None:
         for keyword in node.keywords:
-            if (
-                    keyword.arg is None and
-                    isinstance(keyword.value, ast.Dict) and
+            if all(
```

Comment:
Using `all` as a solution for the W503/W504 warnings -- it is funny how both warnings are presented as solutions to each other via https://www.flake8rules.com/rules/W503.html and https://www.flake8rules.com/rules/W504.html :D